### PR TITLE
fix --path option

### PR DESCRIPTION
**Pull request by rahav-certora**
The --path option is passed as is to the solc compiler as the --allow-paths option.

### CODE REVIEW

File: docs/ref-manual/cli/options.md [hunk 1]
```diff
@@ -264,11 +264,13 @@ When different contracts have to be compiled for different Solidity versions.
 ### `--path`
 
 **What does it do?**  
-Use the given path as the root of the source tree instead of the root of the file system.  
+Passes the value of this option as it to the solc option `--allow-paths`
```
Comment (mdgeorge4153) [inline docs/ref-manual/cli/options.md:267]: `solc` should be code font.

Comment (mdgeorge4153) [inline docs/ref-manual/cli/options.md:267]: "as it" => "as it is"

File: docs/ref-manual/cli/options.md [hunk 2]
```diff
@@ -264,11 +264,13 @@ When different contracts have to be compiled for different Solidity versions.
...
+see [--allow-path specification](https://docs.soliditylang.org/en/v0.8.16/path-resolution.html#allowed-paths)
```
Comment (mdgeorge4153) [inline docs/ref-manual/cli/options.md:268]: capitalize, period.

**Review by mdgeorge4153 — APPROVED** [review]
I added some small nitpicks.  Feel free to merge after you address them

File: docs/ref-manual/cli/options.md (hunk 1 above)
Comment (rahav-certora) [reply docs/ref-manual/cli/options.md:267]: solc is not the right word, changed it to "solidity compiler"

Comment (rahav-certora) [reply docs/ref-manual/cli/options.md:267]: if fact I wanted to write "as is", does that work?

File: docs/ref-manual/cli/options.md (hunk 2 above)
Comment (rahav-certora) [reply docs/ref-manual/cli/options.md:268]: done

**Pull request merged**